assets/base/i18n/en.json

```diff
@@ -1934,6 +1934,8 @@
   "mobile.advanced_settings.reset_button": "Reset",
   "mobile.advanced_settings.reset_message": "\nThis will reset all offline data and restart the app. You will be automatically logged back in once the app restarts.\n",
   "mobile.advanced_settings.reset_title": "Reset Cache",
+  "mobile.advanced_settings.throw_javascript_exception": "Throw JavaScript Exception",
```

Comment:
If this is never going to be seend by the end user, do we really need to translate it? I don’t serme the point